processes/wps_say_hello.py

```diff
@@ -0,0 +1,64 @@
+from pywps import Process, LiteralInput, LiteralOutput, UOM
+from pywps.app.Common import Metadata
+
+import logging
+
+LOGGER = logging.getLogger("PYWPS")
+
+
+class SayHello(Process):
+    """A nice process saying 'hello'."""
+
+    def __init__(self):
+        self.status_percentage_steps = {
+            "start": 0,
+            "complete": 100,
+        }
```

Comment:
We probably don't need this logging status for a test process.

Reply:
I put these in here as part of the `log_handler` test. I could make a comment about that.

Reply:
I think it is fine, I just missed that.